.cirrus.yml

```diff
@@ -96,6 +96,33 @@ linux_qa_task:
       path: "**/target/surefire-reports/*.xml"
       format: junit
 
+mend_scan_task:
+  depends_on:
+    - build
+  # run only on master and long-living branches
+  only_if: $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_TAG == "" && ($CIRRUS_BRANCH == "master" || $CIRRUS_BRANCH =~ "branch-.*" || $CIRRUS_BRANCH =~ "mend-.*")
```

Comment:
nitpick: The comment above doesn't fully match the conditions here - is it actually needed?

Reply:
Good point, I will remove the comment, seems like logic is self-explanatory